one_fm/utils.py

```diff
@@ -1621,6 +1621,7 @@ def create_job_offer_from_job_applicant(job_applicant):
         job_offer.job_applicant = job_app.name
         job_offer.applicant_name = job_app.applicant_name
         job_offer.offer_date = today()
+        job_offer.number_of_days_off = erf.off_days
```

Comment:
We have a complete solution for this line in PR #860. Please check that.

Reply:
Okay. So, let me revert back.